addons/website/static/src/js/editor/widget_link.js

```diff
@@ -26,43 +28,88 @@ weWidgets.LinkTools.include({
     },
     /**
      * Allows the URL input to propose existing website pages.
-     *
+     * Allows the URL input to be filled from document upload.
+     * Automatically creates a link if selected content is a phone number.
+     * 
      * @override
      */
     start: async function () {
         var def = await this._super.apply(this, arguments);
-        wUtils.autocompleteWithPages(this, this.$('input[name="url"]'));
+        this.$urlInput = this.$('#o_link_dialog_url_input');
+        wUtils.autocompleteWithPages(this, this.$urlInput);
         this._adaptPageAnchor();
+        /*
+         * Strips spaces and parenthesis from link text.
+         * If text is a phone number (only composed of digit except for leading +), make the link callable.
+         * e.g +32 4 95 86 85 25 / 04 95 86 85 25 / +32 (0)4 95 86 85 25
+         */
+        const text = this.data.content.replace(/[ )(]/g, '');
```

Comment:
Should this be `this.$urlInput.val()` ?
I tried to create a link but this is taking the selected text (in this case a button which text is `Call Us Now`) instead of the href ?